### ISSUE TITLE
QGIS web application won't open

### ISSUE THREAD

**Issue by larmefatale** [qgis/QGIS#58871]
### What is the bug or the crash?

I have just updated to the lastest LTR QGIS 3.34.11 Prizren.
Prior to that I also updated my Mac OS to mac OS Sequoia 15.0.
When I try and open QGIS for the first time, I do not get the usual prompt but the attached. QGIS just won't open.
![Screenshot 2024-09-25 at 15 16 57](https://github.com/user-attachments/assets/df8a3f27-aad4-461f-ba82-1a630ff711bd)

### Steps to reproduce the issue

1. go to applications folder in mac
2. 'command' + right click to 'Open'
3. the attached appens

### Versions

I can't because I can't open QGIS.

### Supported QGIS version

- [X] I'm running a supported QGIS version according to [the roadmap](https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule).

### New profile

- [ ] I tried with a new [QGIS profile](https://docs.qgis.org/latest/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles)

**Comment by alspringsteen** [comment]
I also had this issue. 

I had to go into System Settings on my Mac (the new Sequoia 15.0), scroll down to Privacy & Security, and then there was a section in there that said QGIS had been blocked and I could click a button to open it anyway.

**Comment by agiudiceandrea** [comment]
@alspringsteen @larmefatale, are the info on https://qgis.org/download/#tips-for-first-launch incorrect?

> After installing QGIS, the first launch attempt may fail due to Apple’s security framework. To enable QGIS, command-click on its icon in your Applications folder and select ‘Open’ in the context menu. A confirmation dialog will display where you need to click the ‘Open’ button again. This only has to be done once.

**Comment by alspringsteen** [comment]
For me, this info on [https://qgis.org/download/#tips-for-first-launch](https://qgis.org/download/#tips-for-first-launch) for Apple was correct until I upgraded my operating system last week to the new macOS Sequoia 15.0.  

It is incorrect now for me on macOS Sequoia 15.0. 

I followed the initial instructions and a confirmation dialog did pop up, but instead of 'Open' it had two options: 'Done' (which closed the dialog), and 'Move to Trash'.

<img width="506" alt="Screenshot 2024-09-26 at 5 36 29 PM" src="https://github.com/user-attachments/assets/0e9fc53c-7e5e-4f1f-97a3-37007780950f">



To solve this, I had to go to System Settings > Privacy & Security and scroll down to the 'Security' section. There was a section there telling me QGIS was prevented from opening but I could choose to open it anyway. 

<img width="712" alt="Screenshot 2024-09-26 at 5 37 30 PM" src="https://github.com/user-attachments/assets/2cc2b10e-1c2f-479e-bf03-24e22799f6dc">



I have only had to do it the once when I first tried to use the new download.

I am not sure how to reach this new dialog in System Settings without first trying to open it the previous way as outlined in the 'tips for launch'.
